app/src/main/res/values-ba/translation.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Remove this file and put the fields in `strings.xml` located in `app/src/main/res/values`